src/components/Buttons/Buttons.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Buttons.css';
+import PropTypes from 'prop-types';
+
+export default function Buttons({ content }) {
```

Comment:
This component should be called Button not Buttons. It displays one Button right?